assets/css/sensei-course-theme/course-navigation.scss

```diff
@@ -2,6 +2,13 @@
 
 
 .sensei-lms-course-navigation {
+	&__modules, &__lessons, &-module__lessons {
+		&, li {
+			list-style: none;
```

Comment:
Checking it in some different themes as a widget, I think it would be worth also adding a `margin: 0` and `padding: 0` here (issue in Storefront, for example).

In a quick check, I think we should just make `.sensei-lms-course-navigation-lesson` a little more specific to override this. ;)

Reply:
I came across those in the themes PR, added more rules there: https://github.com/Automattic/sensei/pull/4775